### ISSUE TITLE
noob question

### ISSUE THREAD

**Issue by 0067sdsm** [morrownr/7612u#16]
hey so i recently got an Alfa AWUS036ACM, so i tried to use this but i dont understand anything, the link for the document that you posted doesnt work, if someone could help me with this that would be very helpfull.

**Comment by morrownr** [comment]
Hi @0067sdsm 

Hey, what's this? I was never a noob!

Just kidding.

Tell us what Linux distro you are using and what you are trying to do. We'll take it from there.

Nick

**Comment by 0067sdsm** [comment]
Hi thank you for the help! I'm trying to use kali Linux, trying to get started hacking wifi (ethically ofc) and i was gonna connect it to my vmware machine but... yeah :I, (was trying checkig by using "lsusb" and the wifite tool)

**Comment by morrownr** [comment]
Hi again @0067sdsm 

> I'm using kali Linux, trying to get started hacking wifi (ethically ofc) 

Good. The adapter should come to life as soon as you plug it in with Kali.

>  i was gonna connect it to my vmware machine

Here is the problem. You have to configure the VM to use the adapter. I would tell you how but I run on bare metal here. If you search the internet using `kali linux vmware usb wifi adapter` you should get many links and videos.

For the long term, you might want to find yourself a laptop that you can dedicate to Kali. It does not have to be new or even very fast for this so you might pick one up for a low price. Just check the hardware to make sure it works well with Linux. If you have questions, you are welcome to come back here.

Nick

**Comment by 0067sdsm** [comment]
Yes, in the vm i can see and do that the adapter is connected, i looks like a little usb symbol but for some reason anything that I try to use it like wifi tools (wifite or "lsusb" the command) just doesn't work

**Comment by morrownr** [comment]
What do you see with the following:

$ iw dev

**Comment by 0067sdsm** [comment]
It shows nothing, like no answer

**Comment by morrownr** [comment]
> It shows nothing, like no answer

Okay. That just means there is no wifi user interface being created in the vm. Since you did not see the adapter with `lsusb` or with `iw dev` then then Kali is not seeing the adapter from the vm.

What you can do to check functionality is use it in Windows. This adapter is plug and play on Win10 and Win11 as well as Linux. 

To continue working the problem: I have no experience with VMWare but I do, sometimes, use Virtualbox. Here is a link to an article about getting usb wifi adapters going in VM's in Virtualbox:

https://null-byte.wonderhowto.com/forum/wifi-hacking-attach-usb-wireless-adapter-with-virtual-box-0324433/

It does not hurt to install Virtualbox so as to play around and see what you can learn. As you are new to Linux and you are working on a subject that is quite complex anyway, do not get in a hurry. Learn one thing at a time and you will get there.

Nick

**Comment by 0067sdsm** [comment]
Hi, yes I am familiar with nullbyte, sadly this article is a not updated, I think I am getting closer to getting it to work tho, thanks!

**Comment by 0067sdsm** [comment]
Yeah i wasnt lol, still not working.
